### PR TITLE
update add gerardi list

### PR DESCRIPTION
**Pull request by LinguList**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [x] refine existing Concepticon concept set mappings
- [x] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

This list is rather long. But the authors based their search for glosses on Concepticon, so I hope it is easy to check the mappings they made.

### CODE REVIEW

**Comment by AnnikaTjuka** [comment]
I can check the mappings tomorrow.

**Comment by LinguList** [comment]
That would be perfect, many thanks, @AnnikaTjuka !

File: concepticondata/conceptlists/Gerardi-2022-711.tsv [hunk 1]
```diff
@@ -0,0 +1,712 @@
...
+Gerardi-2022-711-519	519	STONE	THE_PHYSICAL_WORLD	857	STONE
+Gerardi-2022-711-520	520	STORY	SPEECH_AND_LANGUAGE	2060	STORY
+Gerardi-2022-711-521	521	STRAIGHTEN			
+Gerardi-2022-711-522	522	STRETCH OUT LEGS			
+Gerardi-2022-711-523	523	STRIKE OR BEAT	BASIC_ACTIONS_AND_TECHNOLOGY	2133	STRIKE OR BEAT
+Gerardi-2022-711-524	524	STRONG	BODY	785	STRONG
+Gerardi-2022-711-525	525	STRONG TASTE			
+Gerardi-2022-711-526	526	SUCK	FOOD_AND_DRINK	1421	SUCK
+Gerardi-2022-711-527	527	SUN	THE_PHYSICAL_WORLD	1343	SUN
+Gerardi-2022-711-528	528	SWEAT	THE_BODY
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Gerardi-2022-711.tsv:529]: 126 SWEAT (PERSPIRE)?

File: concepticondata/conceptlists/Gerardi-2022-711.tsv [hunk 2]
```diff
@@ -0,0 +1,712 @@
...
+Gerardi-2022-711-677	677	GROW	SPATIAL_RELATIONS	479	GROW
+Gerardi-2022-711-678	678	HATE	EMOTIONS_AND_VALUES	1607	HATE
+Gerardi-2022-711-679	679	HAVE	POSSESSION	773	HAVE
+Gerardi-2022-711-680	680	HEART OF PALM			
+Gerardi-2022-711-681	681	HIDE	SPATIAL_RELATIONS	2486	HIDE
+Gerardi-2022-711-682	682	HORNED SCREAMER (BIRD)			
+Gerardi-2022-711-683	683	INTESTINES	THE_BODY	1991	INTESTINES
+Gerardi-2022-711-684	684	LARGE BASKET			
+Gerardi-2022-711-685	685	LIE (FALSEHOOD)	EMOTIONS_AND_VALUES	2826	LIE (FALSEHOOD)
+Gerardi-2022-711-686	686	LOOK AT
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Gerardi-2022-711.tsv:687]: 1819 LOOK?

File: concepticondata/conceptlists/Gerardi-2022-711.tsv [hunk 3]
```diff
@@ -0,0 +1,712 @@
...
+Gerardi-2022-711-688	688	MOVE AWAY	BASIC_ACTIONS_AND_TECHNOLOGY	229	MOVE AWAY
+Gerardi-2022-711-689	689	OPEN	SPATIAL_RELATIONS	1156	OPEN
+Gerardi-2022-711-690	690	POND	THE_PHYSICAL_WORLD	2035	POND
+Gerardi-2022-711-691	691	PRICE	POSSESSION	657	PRICE
+Gerardi-2022-711-692	692	PULL UP			
+Gerardi-2022-711-693	693	PUSH AWAY			
+Gerardi-2022-711-694	694	RESULT	BASIC_ACTIONS_AND_TECHNOLOGY	3969	RESULT
+Gerardi-2022-711-695	695	SAKI MONKEY			
+Gerardi-2022-711-696	696	SCRAPE	BASIC_ACTIONS_AND_TECHNOLOGY	1767	SCRAPE
+Gerardi-2022-711-697	697	SEARCH FOR
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Gerardi-2022-711.tsv:698]: 1468 LOOK FOR?

File: concepticondata/references/references.bib [hunk 4]
```diff
@@ -1,3 +1,14 @@
+@article{Gerardi2022,
+  doi = {10.5334/johd.80},
+  url = {https://doi.org/10.5334/johd.80},
+  year = {2022},
+  publisher = {Ubiquity Press,  Ltd.},
```
Comment (AnnikaTjuka) [inline concepticondata/references/references.bib:5]: There seem to be two blank spaces between ", Ltd.".

**Review by AnnikaTjuka — COMMENTED** [review]
I added a few additional mappings. However, many glosses with missing mappings do not have a semantic field, which raises the question of whether we also need to add the semantic field when the mapping is added.

**Comment by LinguList** [comment]
- [ ] Gerardi-2022-711-71	71	BRAZIL NUT (Bertholletia excelsa)	FOOD_AND_DRINK		
- [ ] Gerardi-2022-711-203	203	GIANT RIVER OTTER	ANIMALS	1262	BROTHER
- [ ] Gerardi-2022-711-686	686	LOOK AT			
- [ ] Gerardi-2022-711-697	697	SEARCH FOR		
- [ ] Gerardi-2022-711-124	124	CULTIVATED FIELD	THE_PHYSICAL_WORLD			
- [ ] Gerardi-2022-711-528	528	SWEAT	THE_BODY		
- [ ] Gerardi-2022-711-77	77	BROTHER	KINSHIP		
- [ ] Gerardi-2022-711-464	464	SHINY			
- [ ] There seem to be two blank spaces between ", Ltd.".

Thanks, I'll look into these now. How glad I am that you spotted the BROTHER :)

**Comment by LinguList** [comment]
I will ignore entries missing semantic fields for now.

**Comment by LinguList** [comment]
sorry for the errors, I am fixing them now.

**Comment by AnnikaTjuka** [comment]
I still get the message that the checks have failed although everything looks fine when I run `concepticon test` on the branch.

**Comment by LinguList** [comment]
@xrotwang, the error is due to the new pyconcepticon version that was pushed right now. Should we ignore it when merging? Integrity tests passed all.

**Comment by xrotwang** [comment]
Yes, just merge. I'll fix when running the final tests for 3.0

**Review by AnnikaTjuka — APPROVED** [review]

**Pull request merged**